lib/PuppeteerSharp/Input/KeyDefinitions.cs

```diff
@@ -0,0 +1,1595 @@
+using System.Collections.Generic;
+
+namespace PuppeteerSharp.Input
+{
+    internal static class KeyDefinitions
+    {
+        private static readonly Dictionary<string, KeyDefinition> Definitions = new Dictionary<string, KeyDefinition>
+        {
+            ["0"] = new KeyDefinition
+            {
+                KeyCode = 48,
+                Key = "0",
+                Code = "Digit0"
+            },
+            ["1"] = new KeyDefinition
+            {
+                KeyCode = 49,
+                Key = "1",
+                Code = "Digit1"
+            },
+            ["2"] = new KeyDefinition
+            {
+                KeyCode = 50,
+                Key = "2",
+                Code = "Digit2"
+            },
+            ["3"] = new KeyDefinition
+            {
+                KeyCode = 51,
+                Key = "3",
+                Code = "Digit3"
+            },
+            ["4"] = new KeyDefinition
+            {
+                KeyCode = 52,
+                Key = "4",
+                Code = "Digit4"
+            },
+            ["5"] = new KeyDefinition
+            {
+                KeyCode = 53,
+                Key = "5",
+                Code = "Digit5"
+            },
+            ["6"] = new KeyDefinition
+            {
+                KeyCode = 54,
+                Key = "6",
+                Code = "Digit6"
+            },
+            ["7"] = new KeyDefinition
+            {
+                KeyCode = 55,
+                Key = "7",
+                Code = "Digit7"
+            },
+            ["8"] = new KeyDefinition
+            {
+                KeyCode = 56,
+                Key = "8",
+                Code = "Digit8"
+            },
+            ["9"] = new KeyDefinition
+            {
+                KeyCode = 57,
+                Key = "9",
+                Code = "Digit9"
+            },
+            ["Power"] = new KeyDefinition
```

Comment:
I'm wondering how would the user know what are our available keys...
Maybe it would be great if we could use an enum or a string.
We can create another issue if you think it would get out of scope.

Reply:
maybe we could add something like this - https://github.com/SeleniumHQ/selenium/blob/master/dotnet/src/webdriver/Keys.cs?

Reply:
Yes that could work

Reply:
hmm I noticed that there isn't a clear way to describe some of the properties.
- we can't use the key of each `KeyDefinition` for it's property's name since some of the are invalid names (digits, empty...)
- `KeyDefinition.Code` isn't unique

Reply:
would a simple class that has an entry for each key in https://github.com/kblok/puppeteer-sharp/pull/180/files#diff-7e6afb5424df8f9e8bee32a50727f245L12 be good enough?
I was thinking of creating an overload for that class
```cs
public static class KeyboardExtensions
{
  public static Task TypeAsync(this Keyboard keyboard, Key key, TypeOptions options = null)
    => keyboard.TypeAsync(key.ToString(), options);
}

public class Key
{
  public string Value { get; }
  private Key(string value) => Value = value;

  public static readonly Keys Digit0 = new Keys("0");
  public static readonly Keys Digit1 = new Keys("1");
  public static readonly Keys Digit2 = new Keys("2");
  public static readonly Keys Digit3 = new Keys("3");

  .
  .
  .

  public override string ToString() => Value;
}
```

Reply:
Agreed on the Key. I don't see why it should be an extension method rather than an overload though.